7_deploy_osh/roles/suse-osh-setup/tasks/local-registry-setup.yml

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
Please remove this part of the change (see https://github.com/SUSE-Cloud/socok8s/pull/93)